src/features/components/templates/hero-banners/trade/trade-hero.module.scss

```diff
@@ -22,8 +23,7 @@
         }
 
         @include breakpoints(tablet) {
-            min-block-size: 85svh;
-            inset-block-end: unset;
+            min-block-size: 82.2rem;
```

Comment:
this is gonna cause issues on home page, please check it as well :pray:

Reply:
It's different component..